setup.sh

```diff
@@ -28,6 +28,7 @@ function runCmd ()
 
 function clean ()
 {
+  runCmd "$NPM_BIN_DIR/grunt clean"
```

Comment:
Something's not right here because the shell command immediately following this deletes **more** stuff:
https://github.com/uProxy/uproxy/blob/master/Gruntfile.coffee#L231

I wonder what happened...just general Node.js/NPM weirdness?